packages/components/molecules/f-media-element/src/constants.js

```diff
@@ -11,3 +11,19 @@ export const FONT_SIZE = {
     XL: 'xl',
     XXL: 'xxl'
 };
+
+export const BREAK_POINTS = {
```

Comment:
I don't see where `BREAK_POINTS` is used.

Reply:
thanks have removed